mail_tracking/models/mail_tracking_email.py

```diff
@@ -102,10 +101,10 @@ def _email_score_tracking_filter(self, domain, order='time desc',
     @api.model
     def email_is_bounced(self, email):
         if email:
-            return len(self._email_score_tracking_filter([
+            return self.search_count([
```

Comment:
For what I can see here, we have 2 options:

1. Remove the `_email_score_tracking_filter` method, since it is not being used now.

1. Change the `_email_score_tracking_filter` signature to be `(*args, **kwargs)`, which get passed down to `self.search(*args, **kwargs)`.

   Then, call here to `_email_score_tracking_filter`, we add `count=True` as an argument, which acts just like `search_count`, but makes the count in the DB side, which is also more performant.

I think you should check if any submodule overwrites the method, to make the sane choice.

Reply:
Seems that nobody is using this. I chose to erase the method